docs/contributing.md

```diff
@@ -49,8 +49,10 @@ test suite you will need:
 * Kotlin:
   * `kotlinc`, the [Kotlin command-line compiler](https://kotlinlang.org/docs/command-line.html).
   * `ktlint`, the [Kotlin linter used to format the generated bindings](https://ktlint.github.io/).
-  * The [Java Native Access](https://github.com/java-native-access/jna#download) JAR downloaded and its path
-    added to your `$CLASSPATH` environment variable.
+  * Several JARs downloaded and their path added to your `$CLASSPATH` environment variable:
+    * [Java Native Access](https://github.com/java-native-access/jna#download)
+    * [KotlinX Serialization Core runtime](https://repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-serialization-core-jvm/1.6.3/kotlinx-serialization-core-jvm-1.6.3.jar)
```

Comment:
The latest version of KotlinX Serialization is currently 1.9.0. Since the CI is also using KotlinX Coroutines 1.6.3, whose latest version is also 1.10.2, I chose a version released at the time, when Kotlin 2.0 had not been released yet.

Should we fix the Kotlin version and upgrade the dependencies here?

Reply:
That makes sense to me. Also, here I think it might be worth mentioning that the serialization libs are optional - eg, something like "Optionally, if you use ..., you will need ..."